docs/tutorials/goldenpath.md

```diff
@@ -0,0 +1,497 @@
+1. Edit the `RpcTest.cs` script to match the following.
+
+<details open>
+<summary>Click to show/hide the Code.
+
+</summary>
+
+```csharp
+using MLAPI;
+using MLAPI.Messaging;
+using UnityEngine;
+
+public class RpcTest : NetworkBehaviour
+{
+    private bool firstTime = true;
+
+    [ClientRpc]
+    void TestClientRpc(int value)
+    {
+        if (IsClient)
+        {
+            Debug.Log("Client Received the RPC #" + value);
+            TestServerRpc(value + 1);
+        }
+    }
+
+    [ServerRpc]
+    void TestServerRpc(int value)
+    {
+        if (IsServer)
+        {
+            Debug.Log("Server Received the RPC #" + value);
+            TestClientRpc(value);
+        }
+    }
+    
+    // Update is called once per frame
+    void Update()
+    {
+        if (IsClient && firstTime)
+        {
+            firstTime = false;
+            TestServerRpc(0);
+        }
+    }
+}
+```
+</details>
+1. Save your scene.
+
+<iframe src="https://www.youtube.com/embed/UEYj16CouLE?playlist=UEYj16CouLE&loop=1&&autoplay=0&controls=1&showinfo=0&mute=1"   width="854px"
+        height="480px" className="video-container" frameborder="0" position="relative" allow="accelerometer; autoplay; loop; playlist; clipboard-write; encrypted-media; gyroscope; picture-in-picture"  allowfullscreen=""></iframe>
+
+
+
+### Testing RPCs
+
+1. Select **File > Build and Run**. 
+1. Stop the player 
+1. Launch the client & server together in a terminal as shown in [Testing the command line helper](#testing-the-command-line-helper). 
+1. After a brief delay, the client and server will spawn.  
+1. In the console, you should expect to see the client and server sending RPC messages to each other. 
+1. The client kicks off the exchange in its `Update` call the first time with a counter value of 0.  
+1. It then makes an RPC call to the server with the next value.  The server receives this and calls the client, etc.  In the console view, you will see:
+
+```
+Server Received the RPC #1
+Client Received the RPC #1
+Server Received the RPC #2
+Client Received the RPC #2
+Server Received the RPC #3
+Client Received the RPC #3
+...
+```
+<iframe src="https://www.youtube.com/embed/0mSciPQauyw?playlist=0mSciPQauyw&loop=1&&autoplay=0&controls=1&showinfo=0&mute=1"   width="854px"
+        height="480px" className="video-container" frameborder="0" position="relative" allow="accelerometer; autoplay; loop; playlist; clipboard-write; encrypted-media; gyroscope; picture-in-picture"  allowfullscreen=""></iframe>
+
+
+
+:::note Congrats!
+Congratulations you have learnt how to use some of the basic building blocks of Multiplayer game creation. For more information on the relevant concepts intorduced here please refer to the following sections of the documentaiton:
+
+- [Network variables (client and server-controlled)](../mlapi-basics/networkvariable.md)
+- [Network transforms](../components/networktransform.md)
+- [RPCs](../advanced-topics/message-system/about-rpc.md)
```

Comment:
may want to make this a see also or next steps section with a heading?

Reply:
Done